test/support.js

```diff
@@ -1 +1,54 @@
 import '@agoric/synpress/support/index';
+import { accountAddresses } from './test.utils';
+
+Cypress.Commands.add('addKeys', params => {
+  const { keyName, mnemonic, expectedAddress } = params;
+  const command = `
+    expect -c "
+      spawn agd keys add ${keyName} --recover --keyring-backend=test
+      expect {
+          \\"override\\" {
+              send \\"y\\r\\"
+              exp_continue
+          }
+          \\"Enter your bip39 mnemonic\\" {
+              send \\"${mnemonic}\\r\\"
+              exp_continue
+          }
+      }
+      expect eof
+    "
+  `;
+
+  cy.exec(command).then(({ stdout }) => {
+    expect(stdout).to.contain(expectedAddress);
+  });
+});
+
+Cypress.Commands.add('setOraclePrice', price => {
+  cy.exec(
+    `agops oracle setPrice --keys gov1,gov2 --pair ATOM.USD --price ${price} --keyring-backend=test`,
+  ).then(({ stdout }) => {
+    expect(stdout).to.not.contain('Error');
+    expect(stdout).to.not.contain('error');
+  });
+});
+
+Cypress.Commands.add('createVault', params => {
```

Comment:
these look good. I hope to eventually provide a JS API from the agoric-cli package or similar so this doesn't have to shell out to a JS program.